src/lib/API/api.js

```diff
@@ -425,7 +425,7 @@ export class APIService {
       `${mauticUrl}/form/submit`,
       payload,
       () => log.info('addMauticContact success'),
-      ({ content }, xhr) => log.error('addMauticContact call failed:', content),
+      ({ content }, xhr) => log.error('addMauticContact call failed:', '', null, { content }),
```

Comment:
Could you check which props also having the object comes from Mautic ? Probably there something like error, message etc so we could create an Error object and log it

Reply:
The first argument received in error callback is response object - I can't find some docs/examples of error callback arguments; Also, I tried to emulate errors from mautic and get just empty string. So I will send hole object to the log as extra data

Reply:
ok